models/gday/R/model2netcdf.GDAY.R

```diff
@@ -23,101 +23,100 @@
 ##' @importFrom ncdf4 ncvar_def ncdim_def nc_create ncvar_put nc_close
 
 model2netcdf.GDAY <- function(outdir, sitelat, sitelon, start_date, end_date) {
-  
-  
-  
-
   G_2_KG <- 0.001
   TONNES_PER_HA_TO_G_M2 <- 100
   THA_2_KG_M2 <- TONNES_PER_HA_TO_G_M2 * 0.001
-  
+
   ### Read in model output in GDAY format
   GDAY.output <- read.csv(file.path(outdir, "gday_out.csv"), header = TRUE, sep = ",", skip = 1)
   GDAY.output.dims <- dim(GDAY.output)
-  
+
   ### Determine number of years and output timestep
   days <- as.Date(start_date):as.Date(end_date)
-    year <- strftime(as.Date(days, origin = "1970-01-01"), "%Y")
+  year <- strftime(as.Date(days, origin = "1970-01-01"), "%Y")
   num.years <- length(unique(year))
   years <- unique(year)
   timestep.s <- 86400
-  
+
   ### Loop over years in GDAY output to create separate netCDF outputs
   for (y in years) {
     if (file.exists(file.path(outdir, paste(y, "nc", sep = ".")))) {
       next
     }
-    
+
     ## Subset data for processing
     sub.GDAY.output <- subset(GDAY.output, year == y)
     sub.GDAY.output.dims <- dim(sub.GDAY.output)
-    
+
     ## Setup outputs for netCDF file in appropriate units
     output <- list()
-    
+
     ## standard variables: C-Fluxes
     output[[1]] <- (sub.GDAY.output[, "auto_resp"] * THA_2_KG_M2) / timestep.s
     output[[2]] <- (sub.GDAY.output[, "hetero_resp"] * THA_2_KG_M2) / timestep.s
     output[[3]] <- (sub.GDAY.output[, "auto_resp"] + sub.GDAY.output[, "hetero_resp"] *
-                      THA_2_KG_M2) / timestep.s
+      THA_2_KG_M2) / timestep.s
```

Comment:
It looks as styler doesn't believe in aligning wrapped expressions with their opening paren!

I'm not inclined to fight that and would probably sidestep the issue by reflowing this block as

```
output[[3]] <- (
    sub.GDAY.output[, "auto_resp"]
  + sub.GDAY.output[, "hetero_resp"] * THA_2_KG_M2
  ) / timestep.s
```

... Which now that I type it that way, looks to me like a bug! I wonder if this was actually supposed to be (auto + hetero)*THA_2_KG_M2 / timestep ?

Reply:
I am not sure :confused:

Reply:
Sorry, this wasn't clearly worded nor was it the right PR to post it in -- I was just musing about a corner case in styler; no changes needed in the workflow.